src/plugins/vis_types/timeseries/public/application/components/panel_config/table.tsx

```diff
@@ -91,13 +92,19 @@ export class TablePanelConfig extends Component<
     (name: keyof TimeseriesVisParams) => (e: React.ChangeEvent<HTMLInputElement>) =>
       this.props.onChange({ [name]: e.target.value });
 
+  handleNumberChange =
```

Comment:
Why duplicating `handleNumberChange` here? Seems like https://github.com/elastic/kibana/blob/7c475d8c92bea390c670eab18d96bd0ebe66d3aa/src/plugins/vis_types/timeseries/public/application/components/lib/create_number_handler.ts#L12 can be used

Reply:
Just wondering I see the following comment in lib: 
![image](https://user-images.githubusercontent.com/16915480/173333799-dba33f55-d456-4d8d-b50e-9c7dc2377980.png)

I think because of that here in table we use own handler for text (you san see it above my number handler) instead of handler from lib.

Which way we should use?

Reply:
Hmm, @alexwizp do you know why this TODO is in there?

Reply:
I think it's because such a handler doesn't really fit the React approach, since each call to it will return a new function and cause the child component to re-render.

But this comment was not made by me =)

Reply:
@flash1293 What do you think we should do?

Reply:
Let's leave it like this for now, I don't think it's very important